src/Restaurant.js

```diff
@@ -1,13 +1,23 @@
-import React from "react";
+import React, { useState } from "react";
 
 const Restaurant = () => {
-  const pizzas = 0;
+  const [orders, setOrders] = useState(0);
```

Comment:
Great that you've already started to look at this 🙂 We'll look at this in more detail on Sunday